### DISCUSSION TITLE
Improve search functionality on npmjs.com

### DISCUSSION THREAD

**Discussion by bert-w** [npm/feedback#682] [Website] [answered]
Hi there,

![image](https://user-images.githubusercontent.com/10498595/167951472-e6e89a28-a172-403f-b2a1-1ddd8d9374af.png)
^please get rid of these UI stats. When searching on npmjs, like https://www.npmjs.com/search?q=js , the results are hard to interpret. I'd like to see plain numbers about stars, downloads and latest change date. I think that would be a big improvement over the simplified list we have right now. Hopefully it could help improve the npm landscape in general and decrease the sheer amount of duplication, forks and useless/unused modules. Popular repositories should float to the top. There is a big improvement to be gained here.

In addition, I notice that sometimes you cannot find the package you are looking for because you type a single character wrong (like a dash `-`). The search results dont seem to account for any typos or determine likelyness.

**Comment by monishcm** [comment, marked as answer]
Thank you for your feedback. We will look into improving this feature.

**Comment by astoilkov** [comment]
I'm mentioning a related discussion I created that explores a way to make npm search better. The main idea is to give fewer points for popularity and more points for consistency of commits, releases, and issue responses. The longer the period of consistency, the better.

Discussion — https://github.com/npm/feedback/discussions/718
Proposal — https://astoilkov.com/better-npm-search-proposal
